tracsubtickets/web_ui.py

```diff
@@ -98,10 +100,8 @@ def _append_parent_links(self, req, data, ids):
     def prepare_ticket(self, req, ticket, fields, actions):
         pass
 
-    def get_children(self, parent_id, db=None):
+    def get_children(self, parent_id, db):
```

Comment:
I'd remove the `db` parameter here. Another purpose of context managers is to avoid having to pass the `db` parameter. Context managers provide implicit nesting, even within multiple scopes.